### PR TITLE
✨ Add new server-side vulnerability report

### PR DESCRIPTION
**Pull request by czunker**
cnspec fetches the vulnerability report from upstream and displays the list of vulnerable packages from this report.

The affected commands are `scan` and `vuln`.

### CODE REVIEW

File: apps/cnspec/cmd/vuln.go [hunk 1]
```diff
@@ -59,15 +61,14 @@ var vulnCmdRun = func(cmd *cobra.Command, runtime *providers.Runtime, cliRes *pl
 		log.Fatal().Msg(unauthedErrorMsg)
 	}
 
-	res, err := runtime.Provider.Instance.Plugin.Connect(&plugin.ConnectReq{
```
Comment (czunker) [inline apps/cnspec/cmd/vuln.go]: This is the fix for https://github.com/mondoohq/cnspec/issues/977

Comment (czunker) [reply apps/cnspec/cmd/vuln.go]: This is now also a separate PR: https://github.com/mondoohq/cnspec/pull/979

**Review by imilchev — COMMENTED** [review]
there seem to be a lot of errors here

**Comment by czunker** [comment]
> there seem to be a lot of errors here

Yes, this will only work, after https://github.com/mondoohq/cnquery/pull/2654 is merged and this PR is updated to latest cnquery and go-mondoo.

**Review by imilchev — APPROVED** [review]
LGTM!

**Pull request merged**